medmodels/treatment_effect_estimation/tests/test_treatment_effect.py

```diff
@@ -735,7 +735,9 @@ def test_nearest_neighbors(self):
             TreatmentEffect.builder()
             .with_treatment("Rivaroxaban")
             .with_outcome("Stroke")
-            .with_nearest_neighbors_matching(distance_metric="absolute")
+            .with_nearest_neighbors_matching(
+                distance_metric="euclidean",
+            )
```

Comment:
Could add the full list of subjects here now since we should always get the full result.